app/src/main/java/swati4star/createpdf/fragment/ExceltoPdfFragment.java

```diff
@@ -89,6 +87,8 @@
     @BindView(R.id.recyclerViewFiles)
     RecyclerView mRecyclerViewFiles;
 
+    StringUtils stringUtils = StringUtils.getInstance();
```

Comment:
Shouldn't this be a private variable and initialized in onAttach like rest of the others.

Reply:
Your checkstlye doesn't let me make it private